src/OctoshiftCLI.Tests/gei/Commands/GenerateScriptCommandTests.cs

```diff
@@ -24,7 +25,7 @@ public class GenerateScriptCommandTests
         private readonly Mock<OctoLogger> _mockOctoLogger = TestHelpers.CreateMock<OctoLogger>();
         private readonly Mock<IVersionProvider> _mockVersionProvider = new Mock<IVersionProvider>();
 
-        private readonly GenerateScriptCommand _command;
+        private readonly GenerateScriptCommandHandler _command;
```

Comment:
I didn't notice this in your previous PR but we need to create another test for the handler and move all tests except `Should_Have_Option` from here to the new file.

Reply:
I'll do that in a separate PR, because it's going to generate a giant diff. I'll do one PR and split all the tests into commands vs handlers for all 3 projects.

Reply:
- I think the `Should_Have_Option` is still valuable, because they test the option names, etc. there has been many times  they caught my bugs :) 
- Regarding a separate PR, I agree, we can basically separate the tests in all 3 CLIs.